defender/forta_large_delegations/serverless.yml

```diff
@@ -32,43 +32,56 @@ defender:
 
 functions:
   autotask-1:
-    name: ${self:custom.name} Serverless Autotask
-    path: autotask-1
+    name: Forta Large Delegations Filter
     trigger:
-      type: schedule
-      frequency: 525600000
+      type: webhook
     paused: false
+    path: 'autotask-1'
 
 resources:
   Resources:
     policies:
```

Comment:
The policy is not used by this stack and can be removed from this serverless file.